src/Workspaces/VisualBasic/Portable/Extensions/SyntaxNodeExtensions_SharedWithCodeStyle.vb

```diff
@@ -0,0 +1,120 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Runtime.CompilerServices
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
```

Comment:
This is a new file which can conflict with new file in shared layer which I am adding in the refactoring PR, especially if the code is being merged with another existing extension method file. I feel we are unnecessarily introducing work by creating these small PRs.